packages/core/strapi/lib/services/audit-logs.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const ee = require('../utils/ee');
+
+const createAuditLogsService = (strapi) => {
+  const eventListener = (name, payload = {}) => {
+    if (!name && !Object.keys(payload).length) {
+      throw Error('No name or payload');
+    }
+    // TODO: filter events here
+    // TODO: save events here via provider
+    console.log(`Listened to event ${name} with payload: ${JSON.stringify(payload)}`);
+  };
+
+  const isEnabled = strapi.EE && ee.features.isEnabled('audit-logs');
+
+  return {
+    emitEvent(name, payload) {
```

Comment:
Maybe we should just call this `add` since what we expect once this is called is a new audit log saved to the db

Reply:
That's true, but at the same time "creating an audit log" sounds weird to me, thinking in events seems more natural to me at least

Reply:
I guess I feel the opposite, "emit event" is starting to feel weird to me. If I see that in the codebase I don't get the notion that it is saving something somewhere but just communicating something somewhere and I'll wonder what do I need to do with it next. Whereas if I see `add`  it's clear what's going to happen and I'm not so concerned with how it happens.

Reply:
We'll stick with emitEvent